src/linttest/oop_test.go

```diff
@@ -104,23 +104,6 @@ $_ = WithProps::$int;
 	test.RunAndMatch()
 }
 
-func TestBadModifiers(t *testing.T) {
-	t.Skip("Should be handled by other check, like keywordCase from #138")
-
-	test := linttest.NewSuite(t)
-	test.AddFile(`<?php
-class Foo {
-  /***/
-  PUBLIC Static function f() {}
```

Comment:
Why remove this test?

Reply:
It's was always skipped and issues are covered in keywordCase test.